src/components/arrow/index.ts

```diff
@@ -1,3 +1,4 @@
-import Arrow from "./arrow";
+import Arrow, { ArrowComponentType, ArrowStyle, ArrowProps } from "./arrow";
```

Comment:
```suggestion
export { ArrowComponentType, ArrowStyle, ArrowProps } from "./arrow";
import Arrow from "./arrow";
export default Arrow;
```
not sure if that's better.

Reply:
🤷

Reply:
We can also change the file tree and replace `components/arrow/index.tsx` with `components/arrow.tsx`

Reply:
That sounds great.

Reply:
done in https://github.com/farbenmeer/react-spring-slider/pull/207/commits/21aac613110864868c74633c59ffc75291bcc4d5